tests/test_flask_ui_packages.py

```diff
@@ -496,12 +496,13 @@ def test_package_retire(self, login_func, mock_func):
                 '<li class="errors">No package of this name found.</li>'
                 in output.data)
 
-    @patch('pkgdb2.lib.utils.get_packagers')
+    @patch('pkgdb2.lib.utils')
     @patch('pkgdb2.packager_login_required')
     def test_package_take(self, login_func, mock_func):
         """ Test the package_take function. """
         login_func.return_value = None
-        mock_func.return_value = ['pingou', 'toshio']
+        mock_func.get_packagers.return_value = ['pingou', 'toshio']
```

Comment:
FWIW, ``mock_func`` should probably be renamed to something like ``utils_module`` or ``utils_mod``.